app/com/linkedin/drelephant/spark/fetchers/SparkRestClient.scala

```diff
@@ -143,6 +144,7 @@ object SparkRestClient {
     }
 
     val objectMapper = new ObjectMapper() with ScalaObjectMapper
+    objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
```

Comment:
I think this configuration already exist in the latest SparkRestClient. Check this:

https://github.com/linkedin/dr-elephant/blob/master/app/com/linkedin/drelephant/spark/fetchers/SparkRestClient.scala#L155

Maybe you just need to update your local repo, and only commit the unit test?